dist/stat_data.py

```diff
@@ -285,6 +285,8 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_remove', 'cache overflow table remove calls'),
     CacheStat('cache_lookaside_score', 'cache overflow score', 'no_clear,no_scale'),
     CacheStat('cache_overhead', 'percentage overhead', 'no_clear,no_scale'),
+    CacheStat('cache_page_instantiate_read_lookaside', 'pages read into cache requiring cache overflow entries.'),
+    CacheStat('cache_page_instantiate_read_lookaside_checkpoint', 'pages read into cache by checkpoint requiring cache overflow entries.'),
```

Comment:
The `cache_read_lookaside_checkpoint` is removed and `cache_read_lookaside` is a plan to use whenever a read is satisfied with LAS record instead of page instantiation time, because of this reason, I added two more new parameters just for page instantiation.